views/feed.erb

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+  <head>
+    <title>Home/Chitter</title>
+    <style>
+      body {
+        font-family: "Helvetica Neue", Roboto, "Segoe UI", Calibri, sans-serif;
+        background-color: white;
+        background-image: url("https://wallpaperaccess.com/full/1459046.png")
+      }
```

Comment:
Similar css across views can be saved in an external stylesheet where views could have a link to.

Reply:
agreed.